modules/core/src/main/java/org/apache/ignite/internal/util/GridUnsafe.java

```diff
@@ -1660,6 +1661,12 @@ private static Method newDirectBufferMethod(Object nioAccessObj) {
         return javaVer < 9 ? "sun" : "jdk.internal";
     }
 
+    @NotNull private static boolean directByteBufferLongLongCtor() {
```

Comment:
I suggest to declare a local var in static class initialization and pass it to methods:

```
boolean isDirectBufLongCap = majorJavaVersion(jdkVersion()) >= 21;
```